src/c_cpp_properties.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "optional.h"
+
+struct CCppProperties {
+  std::string cstd;
+  std::string cppstd;
+  std::vector<std::string> args;
```

Comment:
Do we need to support objective-c as well?

Reply:
Didn't [found](https://clang.llvm.org/docs/CommandGuide/clang.html) any mention of obj-c standard